geocity/apps/reports/services.py

```diff
@@ -0,0 +1,48 @@
+from datetime import timedelta
+
+from django.http import FileResponse
+from django.urls import reverse
+from django.utils import timezone
+from knox.models import AuthToken
+
+from .utils import run_docker_container
+
+
+def generate_report_pdf(user, submission_id, form_id, report_id):
+    authtoken, token = AuthToken.objects.create(user, expiry=timedelta(minutes=5))
+
+    url = reverse(
+        "reports:submission_report_content",
+        kwargs={
+            "submission_id": submission_id,
+            "form_id": form_id,
+            "report_id": report_id,
+        },
+    )
+
+    commands = [
+        f"http://web:9000{url}",
```

Comment:
I think the intention is that the `reports` and `pdf` apps aren't coupled, so I think this might not be a good idea. I'm not sure if we can re-use `INTERNAL_WEB_ROOT_URL` reliably; it might be a good issue to raise.